collections/src/main/scala/strawman/collection/SeqView.scala

```diff
@@ -0,0 +1,41 @@
+package strawman
+package collection
+
+import scala.{Int, IndexOutOfBoundsException}
+
+trait SeqView[+A] extends SeqOps[A, View, View[A]] with View[A] {
```

Comment:
could it extends `SeqOps[A, SeqView, SeqView[A]]`?

Reply:
The problem is that if we do that transformation operations have to *always* return a `SeqView`, and sometimes (e.g. `filter`) this is not possible without computing the entire result of the transformation operation, which would defeat the purpose of using views in the first place.